language/en_us/virtualmin.php

```diff
@@ -101,6 +101,7 @@
 $lang['Virtualmin.!error.user_name_valid'] = "The User Name appears to be invalid.";
 $lang['Virtualmin.!error.port_valid'] = "The Port appears to be invalid.";
 $lang['Virtualmin.!error.password_valid'] = "The Password appears to be invalid.";
+$lang['Virtualmin.!error.password_valid_connection'] = "The User Name and Password combination appear to be invalid.";
```

Comment:
"A connection to the server could not be established. Please check to ensure that the Hostname, User Name, Password, and Port are correct."